selfdrive/car/tests/test_docs.py

```diff
@@ -22,6 +27,13 @@ def test_generator(self):
     self.assertEqual(generated_cars_md, current_cars_md,
                      "Run selfdrive/car/docs.py to update the compatibility documentation")
 
+  def test_docs_diff(self):
+    with redirect_stdout(None):
```

Comment:
is this to just get rid of `Dumping car info to...`?

Reply:
Also to get rid of the potential output of print_car_info_diff. If we don't mind the output, I can remove the line

Reply:
the diff should always be none, so should be fine